src/SFA.DAS.Tools.Servicebus.Support.Functions/IoC.cs

```diff
@@ -48,9 +48,8 @@ public static IServiceCollection AddServices(this IServiceCollection services, I
                     policyRegistry.Get<IAsyncPolicy>(Constants.MessageQueueWaitAndRetry)
                 );
             });
-            services.AddTransient<ICosmosInfrastructureService, CosmosInfrastructureService>(s => new CosmosInfrastructureService(configuration));
-            services.AddTransient<ICosmosMessageDbContext, CosmosMessageDbContext>(s => new CosmosMessageDbContext(s.GetRequiredService<CosmosClient>(), s.GetService<IUserService>(), configuration, s.GetRequiredService<ILogger<CosmosMessageDbContext>>(), s.GetRequiredService<ICosmosInfrastructureService>()));
-            services.AddTransient<ICosmosUserSessionDbContext, CosmosUserSessionDbContext>(s => new CosmosUserSessionDbContext(s.GetRequiredService<CosmosClient>(), s.GetRequiredService<ICosmosInfrastructureService>(), configuration));
```

Comment:
ICosmosUserSessionDbContext is required - function app is using GetExpiredUserSessionsAsync

Reply:
Added it back in